### PR TITLE
Relative times, short date format.

### PR DESCRIPTION
**Pull request by JanisE**
- Grammar for relative times (see http://www.uzdevumi.lv/p/latviesu-valoda/11-klase/stilistika-8130/re-68541d02-c66f-4e5b-bd29-ce6de3421c5a).
- Fix singular unit durations for withoutSuffix=true cases (they matched withoutSuffix=false cases).
- The short date format adjusted (see http://likumi.lv/ta/id/219491-dokumentu-izstradasanas-un-noformesanas-kartiba#p-363331).
- Cache already split units, do not split them every time.

### CODE REVIEW

**Comment by ichernev** [comment]
@skakri what do you think?

**Comment by skakri** [comment]
:+1: much better than original implementation. My tenses are way off.

File: src/locale/lv.js [hunk 1]
```diff
@@ -5,23 +5,39 @@
 import moment from '../moment';
```
Comment (skakri) [inline src/locale/lv.js]: @JanisE you should add yourself to authors.

**Comment by ichernev** [comment]
@JanisE looks good. You can add yourself to the authors at the top of the file if you want. I'll merge this in next release.

**Comment by ichernev** [comment]
Merged in 9598a5888d00e63ba0505fa2e419ffa4179b8e5d

**Pull request closed without merging**